### PR TITLE
Port Goblins from Frontier for ghostroles.

### PR DESCRIPTION
**Pull request by widgetbeck**
**Changelog**

:cl:
- add: Ported Goblins from New Frontiers for ghostrole purposes.
- add: New ghostrole, the Goblin Stowaways. Build Goblinhome.
- add: New ghostrole, Sir Gollylad, a noble Goblin Knight.
- change: Cowtools now fit in belts.

### CODE REVIEW

**Comment by widgetbeck** [comment]
that's a deliberate choice. the guidebook entries that exist for goblins at
the moment are direct from Frontier and are for them as a roundstart
species. they'll need custom guidebook entries for imp as they will not be
roundstart, and some of the features they have on Frontier weren't ported

**Comment by widgetbeck** [comment]
this is done now by the way. y'all can merge it whenever

File: Resources/Locale/en-US/_NF/traits/traits.ftl
Comment (hivehum) [inline Resources/Locale/en-US/_NF/traits/traits.ftl:1]: what are the purpose of most of these new trait locale strings? you aren't adding Goblin cant as a trait, right? rest of these are out of scope and otherwise not present in this pr as well

**Comment by widgetbeck** [comment]
They're ported directly from NF. I removed the code to display Goblin Cant
as a selectable trait, but the locale strings are still needed for the game
to not complain about the existence of the goblin accent

**Comment by hivehum** [comment]
that makes sense. what about the other trait locale strings? its like a nothing issue overall, i just think it's a little messy if we have locale strings for traits we dont have implemented

**Comment by widgetbeck** [comment]
ah, i hadn't noticed those. got 'em.

**Comment by hivehum** [comment]
do you want this merged now or do you wanna wait until mq finishes the resprite btw

**Comment by widgetbeck** [comment]
it's good to merge now. It being on main branch will make it easier for MQ to work on it. i have temporarily made the goblins' clothes use the human sprites, so everything looks good enough for a test run.

**Review by hivehum — APPROVED** [review]
🫡
